workers/unity/Packages/io.improbable.gdk.tools/.CodeGenTemplate/CodeGen.sln.DotSettings

```diff
@@ -0,0 +1,97 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```

Comment:
Should this be committed?

Reply:
to ensure we follow our style - yes

Reply:
Is this copied over with the template?

Reply:
if we use relative paths to the settings at the repo root, it'll break when .CodeGenTemplate is copied over